lib/protobuf/generators/field_generator.rb

```diff
@@ -20,7 +20,7 @@ class FieldGenerator < Base
       attr_reader :field_options
 
       def applicable_options
-        @applicable_options ||= field_options.map { |k, v| ":#{k} => #{v}" }
```

Comment:
Addressed with a comment.

(Not applicable to proto field names, but inspect is nice: `:'boom"'.inspect #=> ":\"boom\\\"\""`)